server/src/main/java/org/apache/gravitino/server/web/filter/AccessControlNotAllowedFilter.java

```diff
@@ -43,13 +40,10 @@ public class AccessControlNotAllowedFilter implements ContainerRequestFilter {
   @Override
   public void filter(ContainerRequestContext requestContext) throws IOException {
     requestContext.abortWith(
-        Response.status(
-                SC_METHOD_NOT_ALLOWED,
-                String.format(
-                    "You should set '%s' to true in the server side `gravitino.conf`"
-                        + " to enable the authorization of the system, otherwise these interfaces can't work.",
-                    Configs.ENABLE_AUTHORIZATION.getKey()))
-            .allow(Collections.emptySet())
-            .build());
+        Utils.unsupportedOperation(
```

Comment:
Is it possible to add a UT to cover this change?

Reply:
Added an IT.